lib/LANraragi/Plugin/Metadata/Fakku.pm

```diff
@@ -24,7 +24,7 @@ sub plugin_info {
         type        => "metadata",
         namespace   => "fakkumetadata",
         author      => "Difegue",
```

Comment:
Looks like the `loginfrom` field disappeared during the rebase here.

Reply:
yep, I probably deleted it by mistake when resolving the conflicts